olm_deploy/README.md

```diff
@@ -0,0 +1,49 @@
+# Overview
+
+This directory contains resources related to deploying Metering through the Operator Lifecycle Manager (OLM) using the most up-to-date operators bundle.
+
+## Overview of the registry Dockerfile(s)
+
+Before we can deploy Metering through OLM, we need to create a `CatalogSource` custom resource, which is a way to expose a collection of an operator's packages and channels. The goal here is create a `CatalogSource` that references the newest version of the metering-ansible-operator's metadata.
+
+We have two Dockerfiles that achieve this goal:
+
+- Dockerfile.registry: Creates a registry image, which contains the latest metering-ansible-operator manifest bundle, but adds additional scripts that manipulate the CSV manifests. This image cannot be run as a stand-alone image.
+- Dockerfile.registry.dev: Creates a registry image containing the repository's manifest bundle, loads that bundle in a sqlite database, and exposes that database using a gRPC interface. This is useful for when you want to test Metering with the default image listed in the CSV.
```

Comment:
I'm still not quite getting the scenario this solves? Probably just shows I am ignorant of the OLM process more generally, but can we break it down more starting from "This is useful..."